client/config/environment.js

```diff
@@ -4,12 +4,14 @@ var devHost = 'http://localhost:1337';
 var prodHost = 'http://gallery.hairbrain.io';
 var devS3 = 'https://gallery13-dev.s3.amazonaws.com';
 var prodS3 = 'https://gallery13.s3.amazonaws.com';
+var gaId = 'UA-12392326-1';
 
 module.exports = function(environment) {
   var ENV = {
     modulePrefix: 'client',
     podModulePrefix: 'client/pods',
     environment: environment,
+    isProduction: (this.environment === 'production'),
```

Comment:
Yes we could further simplify those settings as you suggest.  

I had added the `isProduction` property here to reduce the number of times I was hard coding `environment === 'production'`.  Ultimately we just want the ga code to run if we are in production or if we have the debugging enabled based on `ENV.debugGoogleAnalytics` 

So yes, I we could just change those places to look at if `ENV.googleAnalyticsId` is set though and that might simplify things...